MSTest-browserstack/local.conf.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "capabilities": {
+    "os_version": "14",
+    "device": "iPhone 12",
+    "realMobile": "true",
+    "browserstack.local": "true",
+    "build": "browserstack-build-1",
+    "name": "BStack test",
+    "browserstack.source": "mstest:sample"
```

Comment:
Please follow the correct convention, you can check nunit non-sdk branch

Reply:
Done